### ISSUE TITLE
The PHP version graph is weird for dates without installation data

### ISSUE THREAD

**Issue by stof** [composer/packagist#1225]
See https://packagist.org/packages/behat/mink-selenium2-driver/php-stats#1.5 for an example

For dates without data, the charts shows all the versions (having data on some other points) with an equal share. This can be quite confusing when looking at the chart.

**Comment by Seldaek** [comment]
Yeah this is a known issue but IMO it'll require a better/cleaner chart implementation to be fixed. Or perhaps we could copy the previous datapoint for the 0 dates as workaround for now..
